### PR TITLE
Add Hybrid Transport for WebAuthn

### PR DESCRIPTION
**Pull request by alexsxb**
Motivation:

This has already been added beginning of the year to the master branch and would be very helpful in the current 4.x branch and following releases to add better behavior to chrome/safari/iOS users to also be able to use a smartphone with the hybrid flow.

### CODE REVIEW

**Comment by tsegismont** [comment]
@alexsxb thanks for sending this PR.

Is the test present in the main branch?

File: vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java [hunk 1]
```diff
@@ -41,11 +41,6 @@ public String fmt() {
 
   @Override
   public AttestationCertificates validate(WebAuthnOptions options, MetaData metadata, byte[] clientDataJSON, JsonObject attestation, AuthData authData) throws AttestationException {
-    // AAGUID must be null
-    if (!"00000000-0000-0000-0000-000000000000".equals(authData.getAaguidString())) {
-      throw new AttestationException("AAGUID is not 00000000-0000-0000-0000-000000000000!");
-    }
-
```
Comment (tsegismont) [inline vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java]: That doesn't seem related (it wasn't present in the PR for the main branch).

Comment (alexsxb) [reply vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java]: The test is also present. This is actually related as this is needed for apple/google devices since theys return a non-zero AAGUID (see comment in the original commit). So only if set to hybrid and the non 0 AAGUID is allowed, the hybrid method will work in this combination.

Comment (tsegismont) [reply vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java]: Sorry, this is admittedly a non-expert question but, when I look at https://github.com/eclipse-vertx/vertx-auth/pull/670/files, I can't see this change or the test.

That's why I'm asking how it is related

Comment (alexsxb) [reply vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java]: Hi @tsegismont,
Oh sorry yes, my fault, it was a second one: https://github.com/eclipse-vertx/vertx-auth/pull/671

Comment (tsegismont) [reply vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java]: Ok, thanks for clarifying. In this case, can you please create a separate PR for the backport of #671 ?

Comment (alexsxb) [reply vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java]: sure will do that

Comment (tsegismont) [reply vertx-auth-webauthn/src/main/java/io/vertx/ext/auth/webauthn/impl/attestation/NoneAttestation.java]: Thanks for creating #693 

As soon as this one is cleaned-up, I'll go ahead and merge.

**Comment by alexsxb** [comment]
@tsegismont 

Thx, should be cleaned-up now.

**Review by tsegismont — APPROVED** [review]
LGTM
